tenets/codelingo/psr2/elseif-not-else-if/codelingo.yaml

```diff
@@ -3,6 +3,9 @@ funcs:
     type: resolver
     body: |
       function(rawCode) {
+        if (rawCode.substring(0, 7) !== "else if") {
+          return rawCode
```

Comment:
@Daanikus when would this ever happen? We are only using this function on an else with an if child.

Reply:
@BlakeMScurr This is a hack as the following two snippets are represented by the same CLQL query:

```
else {
  if (true) {
  }
}
```

```
else if (true) {
}
```

This check just lets us distinguish between the two.